pkg/api/policies.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+type InferencePropertyPolicies struct {
+	Remote InferenceProviderPolicies `toml:"remote,omitempty"`
+	Public InferenceProviderPolicies `toml:"public,omitempty"`
+}
+
+type InferenceProviderPolicies struct {
+	Enabled *bool `toml:"enabled,omitempty"`
+}
+
+type InferencePolicies struct {
+	// inference providers by property
+	Property InferencePropertyPolicies `toml:"property,omitempty"`
+	// inference providers by name
+	Provider map[string]InferenceProviderPolicies `toml:"provider,omitempty"`
+	// policies for all inference providers
+	InferenceProviderPolicies
+}
+
+type ToolsProviderPolicies struct {
+	Enabled        *bool `toml:"enabled,omitempty"`
+	ReadOnly       *bool `toml:"read-only,omitempty"`
+	NonDestructive *bool `toml:"non-destructive,omitempty"`
+	// Local indicates if the tool cannot connect to a remote MCP server
+	Local *bool `toml:"local,omitempty"`
```

Comment:
Similar to
https://github.com/manusa/ai-cli/pull/61/files#r2316300996

We might want to add some comments clarifying the purpose and the default value in case it's not provided.